src/Sylius/Behat/Page/Shop/Checkout/PaymentPage.php

```
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Shop\Checkout;
+
+use Sylius\Behat\Page\SymfonyPage;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class PaymentPage extends SymfonyPage implements PaymentPageInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getRouteName()
+    {
+        return 'sylius_shop_checkout_payment';
+    }
+    
+    public function nextStep()
+    {
+        $this->getDocument()->pressButton('Next');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function selectPaymentMethod($paymentMethod)
+    {
+        $paymentMethodElement = $this->getElement('payment_method');
+
+        Assert::notNull($paymentMethodElement, sprintf('Could not find "%s" payment method', $paymentMethod));
```

Doesn't ``getElement`` throws ``ElementNotFoundException`` exception rather than return null?